### DISCUSSION TITLE
How to use Custom Vehicle Weapons.

### DISCUSSION THREAD

**Discussion by ghost** [YimMenu/YimMenu#2191] [Q&A]
I saw that there was such a feature in the menu. I thought I could install a new cannon for my laser-p996 jet, but I didn't understand how to do it. How does this feature work, or how can I mount weapons on vehicles?
![yimftr](https://github.com/YimMenu/YimMenu/assets/130869249/06f16ee0-0998-4b2f-956a-7ab181910013)

**Comment by ghost** [comment]
Ok I learned how to use it. Settings are effective on vehicle missiles. However, if the vehicle you are driving uses explosive cannons, then the changes you make to the machine gun will not be effective on the vehicle.

At least it didn't apply to p-996. I thought I could do a nerf bypass, but it didn't work.

**Reply by binded-zz** [reply]
this works great just needs a reset option

**Comment by brakecelerate** [comment]
It does work on Explosive MG, but not Cannon.
